getting_started.md

```diff
@@ -0,0 +1,202 @@
+
+After installing Julia, you can install Flux by running the following command in the Julia REPL:
+
+```julia
+julia> ] add Flux
+```
+
+<br>
+
+
+Alternatively, you can run the following:
+
+```julia
+julia> using Pkg; Pkg.add("Flux")
+```
+
+<br>
+
+Flux provides GPU support. For more information on obtaining GPU support, see [CUDA.jl](https://github.com/JuliaGPU/CUDA.jl) and [Flux documentation on GPU suppoort](https://fluxml.ai/Flux.jl/stable/gpu/).
+
+<br>
+
+## Create your first model 
+
+In this tutorial, you'll create your first machine learning model using Flux. This is a simple linear regression model that predicts an output array `y` from an input array `x`.
+
+<br>
+
+### Step 1: Import Flux
+
+To import Flux add the following:
+
+```julia
+using Flux
+```
+<br>
+
+### Step 2: Create some train data
+For this example, create some random train data `x` and `y` arrays:
+
+```julia
+x = rand(5)
+y = rand(2) 
+```
+<br>
+
+### Step 3: Define your model
+
+Define a simple regression model by defining the following function:
+
+```julia
+model(x) = W*x .+ b
+```
+<br>
+
+Then, set the parameters of the model (`W` and `b`) to some initial random values:  
+
+```julia
+W = rand(2, 5)
+b = rand(2)
+```
+<br>
+
+### Step 4: Define a loss function
+
+A loss function evaluates a machine learning model's performance. In other words, it measures how far the model is from its target prediction. Flux enables you to define your own custom loss function or you can use one of the [Loss Functions](https://fluxml.ai/Flux.jl/stable/training/training/#Loss-Functions-1) that Flux provides. 
+
+For this example, define a custom loss function:
+
+```julia
+function loss(x, y)
+  ŷ = model(x)
+  sum((y .- ŷ).^2)
+end
+```
+<br>
+
+This function computes the model's prediction for the input `x` and returns the loss for the output `ŷ`.
+
+<br>
+
+### Step 5: Set an optimiser
+
+You train a machine learning model by running an optimization algorithm (optimiser) that finds the best parameters (`W` and `b`). The best parameters for a model are the ones that achieve the best score of the `loss` function. Flux provides [Optimisers](https://fluxml.ai/Flux.jl/stable/training/optimisers/) that you can use to train a model. 
+
+Set a classic gradient descent optimiser with learning rate η = 0.1:
+
+```julia
+opt = Descent(0.1)
+```
+<br>
+
+### Step 6: Train your model
+
+Training a model is the process of computing the gradients with respect to the parameters for each data point in the data. At every step, the optimiser updates all of the parameters until it finds a good value for them. In fact, you can write this process as a *for loop*. Notice that before training your model, you need to zip the training data as `data = zip(x, y)`. Also, you need to set `ps = params([W, b])` to indicate that you want the derivatives of `W` and `b`.
+
+
+You can execute the training process of your model as follows:
+
+```julia
```

Comment:
```suggestion
```julia
data = zip(x, y)
ps = params(W, b)
```

Reply:
Thanks for suggesting the change. It's updated :)